yml/apdb.yaml

```diff
@@ -13,13 +13,11 @@ tables:
   - name: name
     "@id": "#metadata.name"
     datatype: text
-    length: 1024
```

Comment:
This change and the next don't seem to fit the stated ticket purpose, but I do see there is a clarification in the PR description.

Reply:
They were not part of this ticket. I just noticed that APDB had a few text columns with lengths, which are now ignored by Felis. So I decided to put them on this branch in a separate commit.